src/profile_manager/tests/test_views.py

```diff
@@ -562,3 +562,21 @@ def test_update_profile__revert_email_to_original(self):
 
         self.test_student1.refresh_from_db()
         self.assertEqual(self.test_student1.email, orig_email)
+
+    def test_update_profile__custom_label_displayed(self):
+        """student profile update view should change instances of "student" to match custom label set in siteconfig"""
+        # Login a student
+        success = self.client.login(username=self.test_student1.username, password=self.test_password)
+        self.assertTrue(success)
+
+        # Change custom_name_for_student to a non-default option
+        config = SiteConfig.get()
+        config.custom_name_for_student = "CustomStudent"
+        config.save()
+
+        # Get a student's profile settings request and assert it contains the custom label in both areas where it should be
+        # ("How your names will be used" section and visible_to_other_students field)
+        # both of these labels are casefolded as a part of the forms/templates they're a part of, so test for casefolded instances
```

Comment:
I see lower() method being used, not casefold() why is casefolding mentioned here?

Reply:
Sorry, I'd confused the wording! Comment updated to closer reflect what's being checked.
`# both of these labels are lowercase as a part of the forms/templates they're a part of, so test for lowercase instances`